blockchain/reorganization_test.go

```diff
@@ -52,9 +52,9 @@ func TestReorganization(t *testing.T) {
 	}
 	defer teardownFunc()
 
-	// Since we're not dealing with the real block chain, disable
+	// Since we're not dealing with the real block chain, remove
 	// checkpoints and set the coinbase maturity to 1.
-	chain.DisableCheckpoints(true)
+	chain.RemoveCheckpoints()
```

Comment:
Oh yeah, that should be possible if the tests don't want to change that mid-testing, I'll take a look